### PR TITLE
Add CentOS/RHEL configuration to the config article

### PR DESCRIPTION
**Pull request by ankushagarwal**

### CODE REVIEW

**Comment by ankushagarwal** [comment]
@moxiegirl @docker/docs-owners

**Comment by moxiegirl** [comment]
@ankushagarwal Seems like we are getting a lot of convergence between your article and:

http://docs.docker.com/articles/host_integration/

http://docs.docker.com/articles/systemd/

What do you think? Would try to like to try and integrate these or would you like me to make suggestions for it?

**Comment by ankushagarwal** [comment]
@moxiegirl : I agree, there is convergence. There are two topics to be covered for each distro `How to configure docker` and `How to run docker (possibly using a process manager)`. I think one nice long article can cover this.

**Comment by moxiegirl** [comment]
Cool. @ankushagarwal you want to try that and resubmit this? :-)

**Comment by ankushagarwal** [comment]
Sure. I'll take some time to consolidate everything into one article. Should have something by tomorrow.

**Comment by moxiegirl** [comment]
Terrific.  Thanks and no worries if you don't hit tomorrow. Later this week also works.

**Comment by SvenDowideit** [comment]
nice +1!

File: docs/sources/articles/configuring.md [hunk 1]
```diff
@@ -1,17 +1,37 @@
...
 
 In a production environment, system administrators typically configure the
 `docker` daemon to start and stop according to an organization's requirements.  In most
 cases, the system administrator configures a process manager such as `SysVinit`, `Upstart`,
-or `systemd` to manage the `docker` daemon's start and stop.
+, `systemd`, etc. to manage the `docker` daemon's start and stop.
+
+### Running the `docker` daemon directly
+ 
+The docker daemon can be run directly using the `-d` option. By default it listens on
```
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: \`docker\`

File: docs/sources/articles/configuring.md [hunk 2]
```diff
@@ -1,17 +1,37 @@
...
+the unix socket `unix:///var/run/docker.sock`
```
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: Unix

File: docs/sources/articles/configuring.md [hunk 3]
```diff
@@ -1,17 +1,37 @@
...
-flags to it directly when you start it.  
+configuration.   
 
 In a production environment, system administrators typically configure the
 `docker` daemon to start and stop according to an organization's requirements.  In most
 cases, the system administrator configures a process manager such as `SysVinit`, `Upstart`,
-or `systemd` to manage the `docker` daemon's start and stop.
+, `systemd`, etc. to manage the `docker` daemon's start and stop.
+
+### Running the `docker` daemon directly
```
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: Tend to avoid formatting in the title.

File: docs/sources/articles/configuring.md [hunk 4]
```diff
@@ -1,17 +1,37 @@
...
+
+    $ docker -d
+    
+    INFO[0000] +job init_networkdriver()
+    INFO[0000] +job serveapi(unix:///var/run/docker.sock)
+    INFO[0000] Listening for HTTP on unix (/var/run/docker.sock)
+	...
+	...
+
+### Configuring the `docker` daemon directly
```
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: Ditto re formatting

File: docs/sources/articles/configuring.md [hunk 5]
```diff
@@ -1,17 +1,37 @@
...
+    INFO[0000] +job serveapi(unix:///var/run/docker.sock)
+    INFO[0000] Listening for HTTP on unix (/var/run/docker.sock)
+	...
+	...
+
+### Configuring the `docker` daemon directly	
+
+
+If you're running the `docker` daemon directly by running `docker -d` instead 
+of using a process manager, you can append the config options to the run
```
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: configuration

File: docs/sources/articles/configuring.md [hunk 6]
```diff
@@ -1,17 +1,37 @@
...
+command directly. Just like the `-d` option, other options can be passed to the `docker`
```
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: to the \`docker run\` command directly.

File: docs/sources/articles/configuring.md [hunk 7]
```diff
@@ -42,12 +58,18 @@ These options :
 
 ## Ubuntu
 
-After successfully [installing Docker for Ubuntu](/installation/ubuntulinux/), you can check the
-running status using Upstart in this way:
+As of `14.04`, Ubuntu uses `Upstart` as a process manager. By default, `Upstart` jobs
+are located in  `/etc/init` and `docker` can be found at `/etc/init/docker/conf`.
```
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: and \`docker\` ...? Think there is a missing word here.

File: docs/sources/articles/configuring.md [hunk 8]
```diff
@@ -42,12 +58,18 @@ These options :
...
+After successfully [installing Docker for Ubuntu](/installation/ubuntulinux/), 
+you can check the running status using `Upstart` in this way:
 
     $ sudo status docker
+    
     docker start/running, process 989
 
+### Running Docker
+
 You can start/stop/restart `docker` using
```
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: the \`docker\` daemon

File: docs/sources/articles/configuring.md [hunk 9]
```diff
@@ -96,3 +118,84 @@ These options :
 7. Verify that the `docker` daemon is running as specified wit the `ps` command.
```
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: with

File: docs/sources/articles/configuring.md [hunk 10]
```diff
@@ -96,3 +118,84 @@ These options :
...
 
 		$ ps aux | grep docker | grep -v grep
+
+### Logs
+
+By default logs for `Upstart` jobs are located in `/var/log/upstart` and the logs for `docker` daemon
```
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: I don't think you need to format Upstart.

File: docs/sources/articles/configuring.md [hunk 11]
```diff
@@ -96,3 +118,84 @@ These options :
...
+    INFO[0000] Loading containers: done.
+    INFO[0000] docker daemon: 1.6.0 4749651; execdriver: native-0.2; graphdriver: aufs
+    INFO[0000] +job acceptconnections()
+    INFO[0000] -job acceptconnections() = OK (0)
+    INFO[0000] Daemon has completed initialization
+
+
+## CentOS / Red Hat Enterprise Linux / Fedora
+
+As of `7.x`, CentOS and RHEL  use `systemd` as the process manager. Also as of `21`, Fedora uses
```
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: extra space after RHEL?

Comment (jamtur01) [inline docs/sources/articles/configuring.md]: As of version 21, Fedora uses...

File: docs/sources/articles/configuring.md [hunk 12]
```diff
@@ -96,3 +118,84 @@ These options :
...
+`systemd` as it's process manager.
```
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: its

File: docs/sources/articles/configuring.md [hunk 13]
```diff
@@ -96,3 +118,84 @@ These options :
...
+
+After successfully installing Docker for [CentOS](/installation/centos/)/[Red Hat Enterprise Linux]
+(/installation/rhel/)/[Fedora](/installation/fedora), you can check the running status in this way:
+
+    $ sudo systemctl status docker
+
+### Running Docker
+
+You can start/stop/restart `docker` using
```
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: the \`docker\` daemon

File: docs/sources/articles/configuring.md [hunk 14]
```diff
@@ -96,3 +118,84 @@ These options :
...
+
+If you want Docker to start at boot, you should also:
+
+    $ sudo systemctl enable docker
+
+
+### Configuring Docker
+
+You configure the `docker` daemon in the `/etc/sysconfig/docker` file on your
+system. You do this by specifying values in a variable. For CentOS 7.x and RHEL 7.x, the name
```
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: host

File: docs/sources/articles/configuring.md [hunk 15]
```diff
@@ -96,3 +118,84 @@ These options :
...
+of the variable is `OPTIONS` and for CentOS 6.x and RHEL 6.x, the name of the variable is
+`other_args`. For this section, we will use CentOS 7.x as an example to configure `docker`.
```
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: to configure the \`docker\` daemon.

File: docs/sources/articles/configuring.md [hunk 16]
```diff
@@ -96,3 +118,84 @@ These options :
...
+2. If you don't have one, create the `/etc/sysconfig/docker` file in your system. 
+
+	Depending on how you installed Docker, you may already have this file.
+
+3. Open the file with your favorite editor.
+
+		$ sudo vi /etc/sysconfig/docker
+		
+4. Add a `OPTIONS` variable with the following options. These options are appended to the
+`docker` daemon's run command.
```
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: to the command that starts the  \`docker\` daemon.

File: docs/sources/articles/configuring.md [hunk 17]
```diff
@@ -96,3 +118,84 @@ These options :
...
+	``` 
+	 OPTIONS=" --dns 8.8.8.8 --dns 8.8.4.4 -D --tls=false -H tcp://0.0.0.0:2375 "
+	```
+	
+These options : 
+
+- Set `dns` server for all containers
+- Enable `-D` (debug) mode 
+- Set `tls` to false
+- Listen for connections on `tcp://0.0.0.0:2375`
```
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: Might want to link to docs that explain these options or add simple explanations.

File: docs/sources/articles/configuring.md [hunk 18]
```diff
@@ -96,3 +118,84 @@ These options :
...
+  
+5. Save and close the file.
+
+6. Restart the `docker` daemon.
+
+ 		 $ sudo service docker restart
+
+7. Verify that the `docker` daemon is running as specified wit the `ps` command.
```
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: with

File: docs/sources/articles/configuring.md [hunk 19]
```diff
@@ -1,17 +1,37 @@
-page_title: Configuring Docker
-page_description: Configuring the Docker daemon on various distributions
-page_keywords: docker, daemon, configuration
+page_title: Configuring and running Docker
+page_description: Configuring and running the Docker daemon on various distributions
+page_keywords: docker, daemon, configuration, running, process managers
 
-# Configuring Docker on various distributions
+# Configuring and Running Docker on various distributions
```
Comment (thaJeztah) [inline docs/sources/articles/configuring.md]: lowercase Running (-> running)

File: docs/sources/articles/configuring.md [hunk 20]
```diff
@@ -1,17 +1,37 @@
...
 
 After successfully installing Docker, the `docker` daemon runs with it's default
```
Comment (thaJeztah) [inline docs/sources/articles/configuring.md]: Not your fault, but could you change "it's" to "its"?

File: docs/sources/articles/configuring.md [hunk 21]
```diff
@@ -1,17 +1,37 @@
...
-configuration. You can configure the `docker` daemon by passing configuration
-flags to it directly when you start it.  
+configuration.   
 
 In a production environment, system administrators typically configure the
 `docker` daemon to start and stop according to an organization's requirements.  In most
 cases, the system administrator configures a process manager such as `SysVinit`, `Upstart`,
-or `systemd` to manage the `docker` daemon's start and stop.
+, `systemd`, etc. to manage the `docker` daemon's start and stop.
```
Comment (thaJeztah) [inline docs/sources/articles/configuring.md]: Not in favour of this change; ", etc." is superfluous here, because the list starts with "such as ..".

Please revert this change.

File: docs/sources/articles/configuring.md [hunk 22]
```diff
@@ -42,12 +58,18 @@ These options :
 
 ## Ubuntu
 
-After successfully [installing Docker for Ubuntu](/installation/ubuntulinux/), you can check the
-running status using Upstart in this way:
+As of `14.04`, Ubuntu uses `Upstart` as a process manager. By default, `Upstart` jobs
```
Comment (thaJeztah) [inline docs/sources/articles/configuring.md]: I think the ubuntu section will have to be split into <= 14.10 and >= 15.04. Afaik, Ubuntu 15.04 switched to `systemd`, which results in people reporting that their configuration is no longer used.

File: docs/sources/articles/configuring.md [hunk 23]
```diff
@@ -96,3 +118,84 @@ These options :
...
 		$ ps aux | grep docker | grep -v grep
+
+### Logs
+
+By default logs for `Upstart` jobs are located in `/var/log/upstart` and the logs for `docker` daemon
+can be located at `/var/log/upstart/docker.log`
+
+    $ tail -f /var/log/upstart/docker.log
+    INFO[0000] Loading containers: done.
+    INFO[0000] docker daemon: 1.6.0 4749651; execdriver: native-0.2; graphdriver: aufs
```
Comment (thaJeztah) [inline docs/sources/articles/configuring.md]: Nit; is AUFS used on a vanilla Ubuntu 14.04 installation, or is manual interaction required to install the AUFS packages?

If AUFS is not the default, this should be changed to what people will actually see ("devicemapper"?)

File: docs/sources/articles/configuring.md [hunk 24]
```diff
@@ -96,3 +118,84 @@ These options :
...
+of the variable is `OPTIONS` and for CentOS 6.x and RHEL 6.x, the name of the variable is
+`other_args`. For this section, we will use CentOS 7.x as an example to configure `docker`.
+
+To configure Docker options:
+
+1. Log into your system as a user with `sudo` or `root` privileges.
+
+2. If you don't have one, create the `/etc/sysconfig/docker` file in your system. 
+
+	Depending on how you installed Docker, you may already have this file.
```
Comment (thaJeztah) [inline docs/sources/articles/configuring.md]: Looks like you're using a <tab> here for indenting, please use spaces

File: docs/sources/articles/configuring.md [hunk 25]
```diff
@@ -96,3 +118,84 @@ These options :
...
+
+3. Open the file with your favorite editor.
+
+		$ sudo vi /etc/sysconfig/docker
```
Comment (thaJeztah) [inline docs/sources/articles/configuring.md]: Same here, should be 8 spaces here in stead of 2 tabs (but check rendering after you've made the change)

File: docs/sources/articles/configuring.md [hunk 26]
```diff
@@ -96,3 +118,84 @@ These options :
...
+		
+4. Add a `OPTIONS` variable with the following options. These options are appended to the
+`docker` daemon's run command.
+
+	``` 
+	 OPTIONS=" --dns 8.8.8.8 --dns 8.8.4.4 -D --tls=false -H tcp://0.0.0.0:2375 "
+	```
```
Comment (thaJeztah) [inline docs/sources/articles/configuring.md]: For consistency, indent the block with 8 spaces instead of the back-ticks.

(I also noticed there's a space before `OPTIONS` that shouldn't be there)

Comment (thaJeztah) [inline docs/sources/articles/configuring.md]: Also, these settings should really be discouraged! This will make the docker daemon listen on *all* interfaces, without TLS enabled.

I don't think we should "promote" this by using it as an example in the documentation. Can we come up with something else to illustrate what's possible?

File: docs/sources/articles/configuring.md (hunk 23 above)
Comment (ankushagarwal) [inline docs/sources/articles/configuring.md]: I am running docker on an Ubuntu 14.04 VM using vagrant with the options `/usr/bin/docker -d --registry-mirror=http://localhost:5000`. I did not specify `aufs` specifically. Can you confirm that `devicemapper` is the default?

File: docs/sources/articles/configuring.md (hunk 26 above)
Comment (ankushagarwal) [inline docs/sources/articles/configuring.md]: Agreed. Will use a secure configuration.

**Comment by ankushagarwal** [comment]
Thanks @thaJeztah and @jamtur01 for your comments. I have addressed all of them in https://github.com/docker/docker/commit/65f9d9133e7bf32025670fd3b7a92f06f5e2dedd

File: docs/sources/articles/configuring.md (hunk 22 above)
Comment (ankushagarwal) [inline docs/sources/articles/configuring.md]: Let me complete this article first and I can add a >= 15.04 section in a
new PR
ᐧ

On Tue, May 5, 2015 at 10:36 PM, Sebastiaan van Stijn <
notifications@github.com> wrote:

> In docs/sources/articles/configuring.md
> <https://github.com/docker/docker/pull/12804#discussion_r29709921>:
>
> > @@ -42,12 +58,18 @@ These options :
> >
> >  ## Ubuntu
> >
> > -After successfully [installing Docker for Ubuntu](/installation/ubuntulinux/), you can check the
> > -running status using Upstart in this way:
> > +As of `14.04`, Ubuntu uses `Upstart` as a process manager. By default, `Upstart` jobs
>
> I think the ubuntu section will have to be split into <= 14.10 and >=
> 15.04. Afaik, Ubuntu 15.04 switched to systemd, which results in people
> reporting that their configuration is no longer used.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/12804/files#r29709921>.
>



-- 
Ankush Agarwal
http://ankush.io
@ankushio <https://twitter.com/ankushio>

*There are 10 types of people in this world. Those who understand binary
and those who don't.*

File: docs/sources/articles/configuring.md (hunk 23 above)
Comment (thaJeztah) [inline docs/sources/articles/configuring.md]: I don't have access to a vanilla Ubuntu install. Perhaps someone else following this PR is able to check?

it's a nit anyway, not a huge deal

**Comment by ankushagarwal** [comment]
@thaJeztah : Let me complete this article first and I can add a >= 15.04 section in a new PR

**Comment by ankushagarwal** [comment]
@moxiegirl @jamtur01 @thaJeztah : I have pushed the final commit for this article and it's ready to go from my end

File: docs/sources/articles/configuring.md [hunk 27]
```diff
@@ -1,17 +1,36 @@
...
-configuration. You can configure the `docker` daemon by passing configuration
-flags to it directly when you start it.  
+After successfully installing Docker, the `docker` daemon runs with its default
+configuration.
 
 In a production environment, system administrators typically configure the
 `docker` daemon to start and stop according to an organization's requirements.  In most
 cases, the system administrator configures a process manager such as `SysVinit`, `Upstart`,
-or `systemd` to manage the `docker` daemon's start and stop.
+, `systemd` to manage the `docker` daemon's start and stop.
```
Comment (thaJeztah) [inline docs/sources/articles/configuring.md]: looks like you forgot to put back the "or"; replace this comma with "or"

File: docs/sources/articles/configuring.md [hunk 28]
```diff
@@ -21,34 +40,35 @@ Some of the daemon's options are:
...
-
-## Direct Configuration
-
-If you're running the `docker` daemon directly by running `docker -d` instead of using a process manager,
-you can append the config options to the run command directly.
 
+Here is a an example of running the `docker` daemon with configuration options:
 
-Here is a an example of running the `docker` daemon with config options:
+    $ docker -d -D --tls=true --tlscert=/var/docker/server.pem --tlskey=/var/docker/serverkey.pem -H tcp://192.168.59.3:2375
```
Comment (thaJeztah) [inline docs/sources/articles/configuring.md]: For TLS, the port-number should be 2376

File: docs/sources/articles/configuring.md [hunk 29]
```diff
@@ -21,34 +40,35 @@ Some of the daemon's options are:
...
+These options :
 
-These options : 
-
-- Enable `-D` (debug) mode 
-- Set `tls` to false
-- Listen for connections on `tcp://0.0.0.0:2375`
+- Enable `-D` (debug) mode
+- Set `tls` to true with the server certificate and key specified using `--tlscert` and `--tlskey` respectively
+- Listen for connections on `tcp://192.168.59.3:2375`
```
Comment (thaJeztah) [inline docs/sources/articles/configuring.md]: 2376 here as well

File: docs/sources/articles/configuring.md [hunk 30]
```diff
@@ -60,39 +80,158 @@ You can start/stop/restart `docker` using
 ### Configuring Docker
 
 You configure the `docker` daemon in the `/etc/default/docker` file on your
-system.  You do this by specifying values in a `DOCKER_OPTS` variable. 
+system.  You do this by specifying values in a `DOCKER_OPTS` variable.
```
Comment (thaJeztah) [inline docs/sources/articles/configuring.md]: Looks like two spaces before "You" (not your fault)

The trailing space after "variable." is intentional, so that "To configure..." is at a new line.

You can either put back the trailing space, or add a blank line before "To configure"

File: docs/sources/articles/configuring.md [hunk 31]
```diff
@@ -60,39 +80,158 @@ You can start/stop/restart `docker` using
...
+2. If you don't have one, create the `/etc/default/docker` file on your host. Depending on how
+you installed Docker, you may already have this file.
 
 3. Open the file with your favorite editor.
 
-		$ sudo vi /etc/default/docker
-		
+    ```
+    $ sudo vi /etc/default/docker
+    ```
```
Comment (thaJeztah) [inline docs/sources/articles/configuring.md]: Why did you change all these code blocks to use back-ticks? Indenting with 4 or 8 spaces should work (4 or 8 spaces depend on the "nesting" level; if you're in a list or not).

I know the "triple back-ticks" are valid, and do the same, but we use the indenting (almost) anywhere.

File: docs/sources/articles/configuring.md [hunk 32]
```diff
@@ -60,39 +80,158 @@ You can start/stop/restart `docker` using
...
-	
-These options : 
-
-- Set `dns` server for all containers
-- Enable `-D` (debug) mode 
-- Set `tls` to false
-- Listen for connections on `tcp://0.0.0.0:2375`
-  
+```
+    DOCKER_OPTS="-D --tls=true --tlscert=/var/docker/server.pem --tlskey=/var/docker/serverkey.pem -H tcp://192.168.59.3:2375"
```
Comment (thaJeztah) [inline docs/sources/articles/configuring.md]: Port 2376 here as well

File: docs/sources/articles/configuring.md [hunk 33]
```diff
@@ -60,39 +80,158 @@ You can start/stop/restart `docker` using
...
+```
+
+These options :
+
+- Enable `-D` (debug) mode
+- Set `tls` to true with the server certificate and key specified using `--tlscert` and `--tlskey` respectively
+- Listen for connections on `tcp://192.168.59.3:2375`
```
Comment (thaJeztah) [inline docs/sources/articles/configuring.md]: 2376 here as well

File: docs/sources/articles/configuring.md [hunk 34]
```diff
@@ -1,17 +1,36 @@
...
+    INFO[0000] +job init_networkdriver()
+    INFO[0000] +job serveapi(unix:///var/run/docker.sock)
+    INFO[0000] Listening for HTTP on unix (/var/run/docker.sock)
+    ...
+    ...
+
+### Configuring the docker daemon directly
+
+If you're running the `docker` daemon directly by running `docker -d` instead
+of using a process manager, you can append the configuration options to the `docker` run
```
Comment (thaJeztah) [inline docs/sources/articles/configuring.md]: I think "run" should be removed here?

**Comment by thaJeztah** [comment]
> Let me complete this article first and I can add a >= 15.04 section in a new PR

Sounds good to me :+1:

I added some notes, please keep in mind that before merging, the commits [should be squashed](https://docs.docker.com/project/work-issue/#pull-and-rebase-frequently)

File: docs/sources/articles/configuring.md (hunk 31 above)
Comment (ankushagarwal) [inline docs/sources/articles/configuring.md]: When I removed backticks from code blocks between list items, they did not
render properly, so I added them back.
ᐧ

On Wed, May 6, 2015 at 5:54 AM, Sebastiaan van Stijn <
notifications@github.com> wrote:

> In docs/sources/articles/configuring.md
> <https://github.com/docker/docker/pull/12804#discussion_r29732865>:
>
> >
> >  3. Open the file with your favorite editor.
> >
> > -		$ sudo vi /etc/default/docker
> > -		
> > +    ```
> > +    $ sudo vi /etc/default/docker
> > +    ```
>
> Why did you change all these code blocks to use back-ticks? Indenting with
> 4 or 8 spaces should work (4 or 8 spaces depend on the "nesting" level; if
> you're in a list or not).
>
> I know the "triple back-ticks" are valid, and do the same, but we use the
> indenting (almost) anywhere.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/12804/files#r29732865>.
>



-- 
Ankush Agarwal
http://ankush.io
@ankushio <https://twitter.com/ankushio>

*There are 10 types of people in this world. Those who understand binary
and those who don't.*

File: docs/sources/articles/configuring.md (hunk 34 above)
Comment (ankushagarwal) [inline docs/sources/articles/configuring.md]: James Turnbull suggested using this in an earlier note.
ᐧ

On Wed, May 6, 2015 at 6:03 AM, Sebastiaan van Stijn <
notifications@github.com> wrote:

> In docs/sources/articles/configuring.md
> <https://github.com/docker/docker/pull/12804#discussion_r29733082>:
>
> > +
> > +The `docker` daemon can be run directly using the `-d` option. By default it listens on
> > +the Unix socket `unix:///var/run/docker.sock`
> > +
> > +    $ docker -d
> > +
> > +    INFO[0000] +job init_networkdriver()
> > +    INFO[0000] +job serveapi(unix:///var/run/docker.sock)
> > +    INFO[0000] Listening for HTTP on unix (/var/run/docker.sock)
> > +    ...
> > +    ...
> > +
> > +### Configuring the docker daemon directly
> > +
> > +If you're running the `docker` daemon directly by running `docker -d` instead
> > +of using a process manager, you can append the configuration options to the `docker` run
>
> I think "run" should be removed here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/12804/files#r29733082>.
>



-- 
Ankush Agarwal
http://ankush.io
@ankushio <https://twitter.com/ankushio>

*There are 10 types of people in this world. Those who understand binary
and those who don't.*

**Comment by ankushagarwal** [comment]
I will keep pushing commits separately to address comments. Was planning to squash them after an LGTM.

File: docs/sources/articles/configuring.md (hunk 34 above)
Comment (jamtur01) [inline docs/sources/articles/configuring.md]: I think I suggested the \`docker run\` command.

**Comment by moxiegirl** [comment]
@ankushagarwal Sorry for the confusion.   Why don't you squash? We can merge this --- if there is any issue in the content after this review, I will carry the fixes in another PR.

**Comment by ankushagarwal** [comment]
@moxiegirl : Rebased and Squashed

**Comment by ankushagarwal** [comment]
@moxiegirl : Can you merge this if it looks good?

**Comment by moxiegirl** [comment]
LGTM

**Comment by moxiegirl** [comment]
@ankushagarwal We require two LGTM from documentation maintainers.  Just mine so far. So, ping @docker/docs-owners can we get a second.  I saw a few minor things but they were super minor and the perfect is the enemy of the great.

This was a really well done contribution.  Thank you so much for your time and effort.

**Comment by ankushagarwal** [comment]
Glad to be a part of the community and looking forward to more meaningful contributions!

**Comment by jamtur01** [comment]
LGTM

**Comment by thaJeztah** [comment]
Oh! Looks like I missed your previous comment. Thanks so much, and happy to see this merged :+1:
